src/resources/SchemaService/SchemaServiceInterfaces.ts

```diff
@@ -29,3 +31,41 @@ export interface SchemaServiceQueryParams {
     passwordGuid?: string;
     username?: string;
 }
+
+export interface ObjectsToGetField {
+    name: string;
+}
+
+export interface ObjectsToGetRelation {
+    name: string;
+    fields: string[];
+    fromField: string;
+    toField: string;
+    toObject: string;
+}
+export interface ObjectsToGetCondition {
+    field: string;
+    operator: string;
+    type: string;
+    values: string[];
+}
+
+export interface ObjectsToGetObject {
+    name: string;
+    conditions: ObjectsToGetCondition[];
+    fields: ObjectsToGetField[];
+    relations: ObjectsToGetRelation[];
+    attachment: string;
+}
+
+export interface ObjectsToGet {
+    objects: ObjectsToGetObject[];
+}
+
+export interface SchemaServiceSource extends SourceModel {
+    objectsToGet: ObjectsToGet;
+}
+
+export interface CreateSchemaSourceModel extends CreateSourceModel {}
+
```

Comment:
Maybe not necessary to define those. I was thinking it would make the code a bit cleaner